src/plugins/legacy/itkDataImage/writers/itkNiftiDataImageWriter.cpp

```diff
@@ -45,6 +45,12 @@ QStringList itkNiftiDataImageWriter::handled() const {
     return s_handled();
 }
 
+QStringList itkNiftiDataImageWriter::supportedFileExtensions() const
+{
+    return QStringList() << ".nii" << ".nii.gz" << ".nia"
```

Comment:
".nii.gz" << ".nii" << ".hdr" << ".nia" << ".img" << ".img.gz";
Is a better order